clarity/src/vm/docs/contracts.rs

```diff
@@ -44,6 +44,8 @@ struct ContractSupportDocs {
     skip_func_display: HashSet<&'static str>,
 }
 
+const STACKS_BOOT_CODE_MAINNET: [(&'static str, &'static str); 0] = [];
```

Comment:
Why not just drop this entirely?

Reply:
Dropped now -- this docs generation now gets invoked from the boot module.